database/go.mod

```diff
@@ -2,21 +2,23 @@ module obot-platform/database
 
 go 1.23.3
 
+replace github.com/gptscript-ai/go-gptscript => github.com/thedadams/go-gptscript v0.0.0-20250219113618-25d959a071ff
```

Comment:
I need https://github.com/gptscript-ai/go-gptscript/pull/93 to merge before we can remove this and merge this PR.